staging/src/k8s.io/apimachinery/OWNERS

```diff
@@ -7,6 +7,7 @@ approvers:
   - sttts
   - liggitt
   - caesarxuchao
+  - apelisse
```

Comment:
This package includes our codec, serializers, conversion, and schema logic.  How about reviewers first and handle feature requests and refactoring in the broader scope first.

Reply:
Anything for managedfield and patches, I'm a strong +1 on you being an approver.

apimachinery/util covers more than that, but if you want it, watch fanout, and promise not to merge anything to util/framer or util/proxy without consultation I'm ok with that.